neqo-http3/src/control_stream_local.rs

```diff
@@ -25,6 +25,18 @@ impl ::std::fmt::Display for ControlStreamLocal {
 }
 
 impl ControlStreamLocal {
+    pub fn new() -> Self {
+        let mut buf = Vec::new();
+        let mut enc = Encoder::default();
+        enc.encode_varint(HTTP3_UNI_STREAM_TYPE_CONTROL);
+        buf.append(&mut enc.into());
```

Comment:
You don't need `buf` here.  The return value from `enc.into()` is a vector.  Or you could just use `vec![u8::try_from(HTTP3_UNI_STREAM_TYPE_CONTROL).unwrap()]` and save a lot of work.